### PR TITLE
Fixes for MS Visual Studio 2013

### PR DESCRIPTION
**Pull request by magro11**
Dear PCL development team,

this pull request contains three fixes for a proper compilation under MS Visual Studio 2013 if the surface_nurbs project is enabled. It would be great if you could merge this.

Thanks.
Marc

### CODE REVIEW

File: examples/features/example_difference_of_normals.cpp [hunk 1]
```diff
@@ -184,7 +184,8 @@ int main (int argc, char *argv[])
 	range_cond->addComparison (pcl::FieldComparison<PointOutT>::ConstPtr (new
 	pcl::FieldComparison<PointOutT> ("curvature", pcl::ComparisonOps::GT, threshold)));
 	// build the filter
-	pcl::ConditionalRemoval<PointOutT> condrem (range_cond);
+    pcl::ConditionalRemoval<PointOutT> condrem;
+    condrem.setCondition(range_cond);
```
Comment (VictorLamoine) [inline examples/features/example_difference_of_normals.cpp]: Can you please follow the [style guide](http://pointclouds.org/documentation/advanced/pcl_style_guide.php)?

There is an extra space at the beginning of these two lines.
There should be a space before the opening parenthesis.

**Comment by VictorLamoine** [comment]
Please fix style accordingly to my comment and squash your commits, it will be ok to merge after that!
Thanks for the contribution.

**Comment by magro11** [comment]
The misalignment of the first level indentation were inserted since the original file contained tabs in the first level. I replaced now every tab by two spaces each. Hope that this version is now OK.

**Comment by VictorLamoine** [comment]
No :smile: 
You modified indentations from code you did not modify, we don't want that.

It seems you did no other modifications than indentations in `examples/features/example_difference_of_normals.cpp`; please do not include this file in the pull request. (if you have done modifications, please include them but don't change indentation).

`examples/segmentation/example_cpc_segmentation.cpp` is ok :+1:

**Comment by magro11** [comment]
Ok, but then I have to adapt my identation level to the wrong style of the file since the file contained tabs as indentation. You wanted me to add two spaces as first identation in the lines I changed (in total two lines). But if I add two spaces and the rest of the file are tabs, this is also not right. Should I place tabs now in order to be consistent to the wrong style of the file or how do you want me to align my changed lines? (All this only belongs example_difference_of_normals.cpp only)

**Comment by VictorLamoine** [comment]
@taketwo Should we fix the code style in a separate commit?

**Comment by jspricke** [comment]
Yes, fix the style in one commit and put your changes in a new one.

**Comment by magro11** [comment]
Are my changes now correct or is there still something to do from my side?

**Comment by jspricke** [comment]
Can you please squash the commits, to review them?

**Comment by jspricke** [comment]
I've fixed the commits and pushed them into master. Thanks for the fixes!

**Pull request closed without merging**